src/encoding/uleb128.rs

```diff
@@ -19,7 +18,7 @@ pub fn decode(values: &[u8]) -> Result<(u64, usize), Error> {
 
         shift += 7;
     }
-    Ok((result, consumed))
+    (result, consumed)
```

Comment:
the reason this is an error atm is because line 11 has a `panic` on it - it needs to be converted to an error (I left it as panic for a long time, but the error is still relevant as this op is fallible in nature)

Reply:
Alright. I reverted those changes. :+1: